content/recipes/terminus.md

```diff
@@ -1,114 +1,137 @@
 ```
 
-> info **Hint** If done correctly, Nest will expose the defined health check(s), which are reachable through a GET request to the defined route. For example `curl -X GET 'http://localhost:3000/health'`
+The interface of this response object can be accessed from the `@nestjs/terminus` package with the `HealthCheckResult`-interface.
+
+|           |                                                                                                                                                                                             |                                      |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+| `status`  | In case any health indicator failed the status will be `error`. If the NestJS app is shutting down but still accepting HTTP requests, the health check will have the `shutting_down` status. | `'error' \| 'ok' \| 'shutting_down'` |
+| `info`    | Object containing information of each health indicator which is of status `up`, or in other words "healthy".                                                                                | `object`                             |
+| `error`   | Object containing information of each health indicator which is of status `down`, or in other words "unhealthy".                                                                            | `object`                             |
+| `details` | Object containing all information of each health indicator                                                                                                                                  | `object`                             |
 
 #### Custom health indicator
 
-In some cases, the predefined health indicators provided by `@nestjs/terminus` do not cover all of your health check requirements. In this case you can set up a custom health indicator according to your needs.
+In some cases, the predefined health indicators provided by `@nestjs/terminus` do not cover all of your health check requirements. In that case, you can set up a custom health indicator according to your needs.
 
-Let's get started by creating a service which will represent our custom health indicator. To get a basic understanding how a health indicator is structured, we will create an example `DogHealthIndicator`. This health indicator should have the state "up", if every `Dog` object has the type `goodboy`, otherwise it will throw an error, which then the health indicator will be seen as "down".
+Let's get started by creating a service that will represent our custom indicator. To get a basic understanding of how an indicator is structured, we will create an example `DogHealthIndicator`. This service should have the state "up" if every `Dog` object has the type `goodboy`. If that case is not satisfied it then should throw an error.
```

Comment:
state "up" -> state `'up'` (maybe?)

`goodboy` -> `'goodboy'`

If that case is not satisfied
->
If that condition is not satisfied

then should throw
->
then it should throw

Reply:
Resolved, thanks <3